base/stream.jl

```diff
@@ -168,7 +182,6 @@ mutable struct TTY <: LibuvStream
     lock::ReentrantLock
     throttle::Int
     @static if Sys.iswindows(); ispty::Bool; end
-    TTY() = TTY(Libc.malloc(_sizeof_uv_tty), StatusUninit)
```

Comment:
Just curious: why does `PipeEndpoint` have a 0-arg constructor that calls malloc, but not `TTY`?